src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerProxyFacetImpl.java

```diff
@@ -61,6 +62,18 @@ public ComposerProxyFacetImpl(final ComposerJsonProcessor composerJsonProcessor)
     this.composerJsonProcessor = checkNotNull(composerJsonProcessor);
   }
 
+  @Nullable
+  @Override
+  protected Content fetch(Context context, Content stale) throws IOException {
+    try {
+      return super.fetch(context, stale);
+    }
+    catch (NonResolvableProviderJsonException e) {
+      log.debug("Composer provider URL not resolvable: {}", e.getMessage());
```

Comment:
Question: What's the reason for this being debug and not error?

Reply:
> Question: What's the reason for this being debug and not error?

Because it's not always an error, for example in the case that a proxy repo is being queried before a hosted repo that _does_ contain an artifact; in that case the exception path is actually quite common, much like the npm implementation (in that it has to go remote to see if there's even a file containing the download url, and we have to have a way to indicate that within the confines of the formats framework we have). 

See https://github.com/sonatype-nexus-community/nexus-repository-composer/commit/95a20b566a540126c33b10ede3c95758bcf7f2f9 and the comments around https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/14#issuecomment-382306591 and https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/14#issuecomment-382758604 for details.